code/modules/projectiles/guns/energy/cell_loaded_vr/ml3m_cells.dm

```diff
@@ -289,10 +289,10 @@
 
 /obj/item/projectile/beam/medical_cell/omni3/on_hit(var/mob/living/carbon/human/target)
 	if(istype(target, /mob/living/carbon/human))
-		target.adjustBruteLoss(-10)
-		target.adjustFireLoss(-10)
-		target.adjustToxLoss(-10)
-		target.adjustOxyLoss(-60)
+		target.adjustBruteLoss(-20)
+		target.adjustFireLoss(-20)
+		target.adjustToxLoss(-20)
+		target.adjustOxyLoss(-120)
```

Comment:
Doubling every value is a really lazy way of balancing it and making Omni 3 not only heal **120** oxyloss but also effect corpses is insane and trivializes the already one-track state of medical.
For perspective,  this means that a full mag of omni 3 will heal _**420 DAMAGE**_ on **CORPSES** for basically no cost.

Additionally, on the topic of roundstart corpse healing- corpsemend exists for a reason. Realistically, yes, there are a lot of methods for everything in medical but truly all you need in medical (on non-antag servers ie this one) is chems. Osteo is easy to come by and myelamine is ordered from cargo every shift. Even if you don't get osteo, bonemend surgery is easy. 

I don't understand the idea that you want more options, but then are making the medigun overly good. Maybe _you_ will use the different options correctly, but what everyone else is going to do is use the easiest option so they can go back to their scene or whatever they are doing. 

Also, to really show how good the medigun is in it's current state, I kept someone alive during replicant organ transplant with an unupgraded medigun with OMNI 1, sans immunosuprezene. That means spikes of 250 toxins and oxyloss that were suppressed. If it can already handle burst damage in excess of 250, it's powerful enough.

![moth1](https://user-images.githubusercontent.com/65516417/99954519-e9518700-2d50-11eb-8ec2-3b43b0a533f3.png)

Reply:
...Did you not see the comment above, where I mentioned that the cells would be changed to not heal corpses anymore? Other than corpsemend, of course.

Also..chances are, something else was being done to the patient to actually heal them. The base Omni cell would heal 2.5  toxin per shot and could only heal 10 toxin before having to be recharged. Your story just doesn't add up. 

As for the "lazy method" of rebalancing, the reason behind it is because these values were the original plan in the first place when the ML3M was being made. The healing values were all cut in half in its debut PR. 

Medical starts with a base brute cell, burn cell, and stabilizer. All it can do is heal a little oxy, heal 40 brute damage, and heal 40 burn damage. 

And guess what... it takes 10u bicaridine, or kelotane, to do that. Less, with advanced aid. Both methods are incredibly easy, and doctors are more likely to have chems onhand than the ML3M, which there's only one of (other than the cmo's). If you don't get upgraded cells (and a bunch of them) then you still can't do much with it. Chems are still more powerful and convenient.

...also I don't care if someone "finishes up quickly and goes back to their scene". Like.. dude they paused their shit to handle the job, idgaf that they used the fastest method available. 

Any other issues?

Reply:
Also, what organ were you replacing?

Reply:
1. Yes, I did see the comment. I was adding to the existing conversation.
2. The patient was sedated by soporific and was also an SSD sleeve and nothing else. 
3. I see why it was nerfed.
4. Omni is available roundstart and I have NO DOUBT that should it be this powerful it would be made each round. Additionally, the starting cells are better than omni with some judgement.
5. Yes, chems are in-fact the correct option for almost all situations. But putting 10u of a chem AND shooting the medigun is an incredible method of healing even without this change.
6. I don't really know what was meant by this but for most people having to pause their scene is worse than dying. (for reasons I frankly don't understand but it is what it is).

7. These ones. 
```	
	has_organ = list(
		O_HEART =	/obj/item/organ/internal/heart/replicant/rage,
		O_LUNGS =	/obj/item/organ/internal/lungs/replicant/mending,
		O_VOICE =	/obj/item/organ/internal/voicebox/replicant,
		O_LIVER =	/obj/item/organ/internal/liver/replicant,
		O_KIDNEYS =	/obj/item/organ/internal/kidneys/replicant,
		O_BRAIN =	/obj/item/organ/internal/brain/replicant,
		O_EYES =	/obj/item/organ/internal/eyes/replicant,
		O_AREJECT =	/obj/item/organ/internal/immunehub/replicant,
		O_VRLINK =	/obj/item/organ/internal/brainmirror,
		O_STOMACH =	/obj/item/organ/internal/stomach,
		O_INTESTINE =	/obj/item/organ/internal/intestine
		)
```
All of which triggered rejection, resulting in an incredible amount of toxins each time.

Reply:
Bogen basically told me the deets. The only reason the omni cell worked was because the patient was within 7 damage points of death. This does not speak to the strength of the cell, but rather how razor-thin the line was between living and dying. So... yeah. 

Doesn't exactly serve as a reason to not increase the values.

Reply:
After I got home from work, I replicated the experiment without the omni cell and pulled it off without a hitch, so I can confirm that while it may have been useful, it could have been done without.